### PR TITLE
[Modular] GAGS Version of the Flannel Shirt

### PR DESCRIPTION
**Pull request by Jureiia**
## About The Pull Request
Adds a GAGS version of the flannel shirts, leaves the originals in for now.
Available in the loadout and Clothesmate vendor.

## How This Contributes To The Skyrat Roleplay Experience
More options, More colour.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
6tdDCRjuIT
qNp4gEJ0nr
dreamseeker_j7RnbYwhKR
dreamseeker_TUjMRWkfwT
dreamseeker_gZt8X7aJuJ

</details>

## Changelog

:cl:
add: Recolourable Flannel Shirt!
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Is this using the correct GAGS palette? It looks really flat

**Comment by Jureiia** [comment]
> 

Yes, it's using the same palette I used when doing the hawaiian shirt, with an extra shade or two added in to make it even less flat than it would be.

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 1]
```diff
@@ -112,3 +112,11 @@
...
 	name = "brown flannel jacket"
 	icon_state = "flannel_brown"
+
+/obj/item/clothing/suit/toggle/jacket/flannel/gags
+	name = "flannel shirt"
+	icon_state = "flannelgags"
+	greyscale_config = /datum/greyscale_config/flannelgags
+	greyscale_config_worn = /datum/greyscale_config/flannelgags/worn
+	greyscale_colors = "#a61e1f"
+	flags_1 = IS_PLAYER_COLORABLE_1
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm]:
```suggestion
	flags_1 = IS_PLAYER_COLORABLE_1

```

**Review by GoldenAlpharex — APPROVED** [review]
Seems good.

**Pull request merged**